docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+        - [Configure Transaction Allow List](#configure-transaction-allow-list)
+  - [Deploying the Subnet](#deploying-the-Subnet)
+- [Understanding the Genesis File](#understanding-the-genesis-file)
+    - [Config](#config)
+      - [`chainId`](#chainid)
+      - [Hardforks](#hardforks)
+    - [Fee Config](#fee-config)
+      - [`gasLimit`](#gaslimit)
+      - [`targetBlockRate`](#targetblockrate)
+      - [`minBaseFee`](#minbasefee)
+      - [`targetGas`](#targetgas)
+      - [`baseFeeChangeDenominator`](#basefeechangedenominator)
+      - [`minBlockGasCost`](#minblockgascost)
+      - [`maxBlockGasCost`](#maxblockgascost)
+      - [`blockGasCostStep`](#blockgascoststep)
+    - [Validator Fee Recipient](#validator-fee-recipient)
+    - [Precompiles](#precompiles)
+      - [`contractDeployerAllowListConfig`](#contractdeployerallowlistconfig)
+      - [`contractNativeMinterConfig`](#contractnativeminterconfig)
+      - [`txAllowListConfig`](#txallowlistconfig)
+    - [Genesis Block](#genesis-block)
+      - [`nonce`](#nonce)
+      - [`timestamp`](#timestamp)
+      - [`extraData`](#extradata)
+      - [`gasLimit`](#gaslimit-1)
+      - [`difficulty`](#difficulty)
+      - [`mixHash`](#mixhash)
+      - [`coinbase`](#coinbase)
+      - [`parentHash`](#parenthash)
+      - [`gasUsed`](#gasused)
+      - [`number`](#number)
+      - [`airdropHash` `baseFeePerGas` `airdropAmount`](#airdrophash-basefeepergas-airdropamount)
+    - [Native Token Allocation](#native-token-allocation)
+- [Creating a Custom Genesis File](#creating-a-custom-genesis-file)
+  - [Setting the Chain Id](#setting-the-chain-id)
+  - [Configuring Fees and Gas](#configuring-fees-and-gas)
+  - [Allocating Native Tokens](#allocating-native-tokens)
+  - [Minting Native Tokens](#minting-native-tokens)
+  - [Restricting Contract Deployers](#restricting-contract-deployers)
+- [Creating and Deploying the Subnet Using the Custom Genesis File](#creating-and-deploying-the-Subnet-using-the-custom-genesis-file)
+- [Deploying the Game Contract](#deploying-the-game-contract)
+  - [Metamask Configuration](#metamask-configuration)
+  - [Metamask Gas Configuration](#metamask-gas-configuration)
+  - [Deploying the Contract](#deploying-the-contract)
+    - [NativeMinterInterface.sol](#nativeminterinterfacesol)
+    - [Game.sol](#gamesol)
+    - [Compiling and deploying](#compiling-and-deploying)
+  - [Setting Game Contract as a Minter](#setting-game-contract-as-minter)
+  - [Play the Game](#play-the-game)
+- [Conclusion](#conclusion)
+  - [Resources](#resources)
+
+## Prerequisites
+
+### Metamask
+You need to have [Metamask](https://metamask.io/) extension installed on your browser.
+
+### Avalanche CLI
+
+Create a folder for Avalanche CLI.
+
+```bash
+$ mkdir -p ~/avalanche
+```
+
+Go to that directory and download the Avalanche CLI.
+
+```bash
+$ cd ~/avalanche
+$ curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-cli/main/scripts/install.sh | sh -s
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Dollar signs used before commands without showing output.

> MD014/commands-show-output: Dollar signs used before commands without showing output

[More info: MD014](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md014---dollar-signs-used-before-commands-without-showing-output)

Currently if a dev just copies the command, including the `$`, then the command will fail when they paste it into the terminal:

> $ mkdir -p ~/avalanche
> zsh: command not found: $

Instead of showing the return output just remove the `$` from the beginning of the command.